src/pages/PeoplePage/PeopleTable.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Persone from '../../components/Persone';
+import Person from '../../components/Person';
 import { IPerson } from '../../types';
```

Comment:
Please change name of interface to Person.
Also rename Person.tsx to something like PersonComponent.tsx, so it will be clear that it's component
![image](https://user-images.githubusercontent.com/96409563/196113163-6096edd6-08bb-415b-9a94-b43fc5056263.png)